server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsDialect.java

```diff
@@ -175,12 +163,12 @@ private void processTextTransformation(
         .forEach(
             p -> {
               if (copybookLevel > 0 && p.getRight() != null) {
-                CopyIdmsAdjustmentProcessor copyIdmsAdjustmentProcessor = new CopyIdmsAdjustmentProcessor(
+                CopyIdmsAdjustmentProcessor copyIdmsAdjustmentProcessor = new CopyIdmsAdjustmentProcessor(copyNode,
                         currentDocumentMap.getUri(), copybookLevel, firstLevel, p, messageService);
-                processors.add(new ProcessorDescription(CopyNode.class, ProcessingPhase.TRANSFORMATION, copyIdmsAdjustmentProcessor));
```

Comment:
This resulted in an accumulation of errors if we close and opened the same file. Resulting in the same error multiple times. Trying to get rid of my previous sin here 🗡️